### DISCUSSION TITLE
Syncing with Google Drive or One Drive

### DISCUSSION THREAD

**Discussion by dinbtechit** [isar/isar#1027] [Ideas]
It would be nice if IsarDB file can be stored in a Google Drive or One Drive and synced up periodically

**Comment by mikekosulin** [comment]
You can implement cloud sync using exportJson/importJson query functions along with any cloud storage package and worker to run periodically backups in isolate.

Job management:
https://pub.dev/packages/worker_manager

For GoogleDrive and OneDrive:
https://pub.dev/packages/cloud9

**Reply by dinbtechit** [reply]
Wow this is pretty nifty! Not sure why it does not show the option to make the best answer.

**Reply by joneldominic** [reply]
Would you mind sharing your experience implementing this one? Does it work seamlessly?
